apps/docs/components/reference/RefFunctionSection.tsx

```diff
@@ -114,7 +114,13 @@ const RefFunctionSection: React.FC<IRefFunctionSection> = (props) => {
                 >
                   {item.examples &&
                     item.examples.map((example, exampleIndex) => {
-                      const exampleString = ''
+                      const exampleString =
+                        '' +
+                        (example.code &&
+                          example.code
+                            .trim()
+                            .replace(/^```.*/, '')
+                            .replace(/```$/, ''))
```

Comment:
I don’t think so, not without complicating it quite a bit.

I don’t know for sure, but the suggested code looks like it will only replace the first line with an empty space. The last line won’t be replaced anymore, which isn’t desired.

Reply:
Using the combined regex, it doesn't seem like it matches the markdown code at all https://regex101.com/r/3TN4II/1.
Whereas the two separate regexes do work, https://regex101.com/r/HzSn28/1 and https://regex101.com/r/Ka3quQ/1.

Reply:
@gregnr Just want to check back on this and see if this is fine as is.